boto/ec2/connection.py

```diff
@@ -4423,6 +4424,11 @@ def copy_image(self, source_region, source_image_id, name=None,
             params['Description'] = description
         if client_token is not None:
             params['ClientToken'] = client_token
+        if encrypted is not None:
+            params['Encrypted'] = 'true' if encrypted else 'false'
+
+            if kms_key_id is not None:
```

Comment:
Ok. I used the same logic as create_volume above. Wanted to prevent the user from shooting themselves in the foot, but agree that this just gives them a silencer!

Updated the PR and rebased off the develop branch. Please take a look!